Makefile

```diff
@@ -98,13 +98,15 @@ venv:
 	# Create a virtual environment which can be used to run the build package.
 	python3 -m venv venv --prompt="CodeChecker venv" && \
 		$(ACTIVATE_RUNTIME_VENV) && \
+		pip3 install wheel && \
```

Comment:
Why is this better than having the package specified in the `requirements.txt`?

Reply:
I believe, @gamesh411 had this answer for this question from earlier:

> The wheel package is only used for building the other two packages, and thus will not contribute to the runtime code directly, so there are no API contracts to uphold/break. So at build time, an up-to-date version of the package seems like a good idea to me.

However, I would also agree to put dependencies in the `requirements.txt` of the `venv_dev` target. This target is dedicated for dependencies during development and not during usage.